build-lab.sh

```diff
@@ -27,6 +27,10 @@ sudo apt-get install -y tree
 
 echo -e "${LINE_ASCII_CONSOLE}\n"
 
+echo "gem: --no-ri --no-rdoc" >> ${lab}/.gemrc
+
+export CHEF_LICENSE="accept-silent"
```

Comment:
Question: are we planning on moving to cinc auditor instead of inspec at some point for the class?

Reply:
Anticipate that some students will forget or not notice that they are supposed to `source` this script instead of running it directly.  If they fail to `source` then neither the `lab` alias nor the `CHEF_LICENSE` environment variable will be set, so behavior will be different for these students.  I believe we can check for `source` and issue a warning and exit if the script was executed directly.  I'll look into it...